ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPackageManagementService.scala

```diff
@@ -123,7 +127,7 @@ private[apiserver] final class ApiPackageManagementService private (
         )
 
       val response = for {
-        dar <- Future.fromTry(decodeAndValidate(darInputStream))
```

Comment:
Not strictly related to your PR, but where is this darInputStream closed? It doesn't seem to be a AutoCloseable.